_includes/_footer.html

```diff
@@ -40,6 +41,9 @@ <h2 class="ds-h4">Additional resources</h2>
           <li><a target="_blank" rel=noopener href="https://www.cms.gov/About-CMS/Agency-Information/Aboutwebsite/NoFearAct.html">No Fear Act</a></li>
           <li><a target="_blank" rel=noopener href="https://www.medicare.gov/about-us/plain-writing/plain-writing.html">Plain Writing</a></li>
           <li><a target="_blank" rel=noopener href="https://www.usa.gov/">USA.gov</a></li>
+          <li><a target="_blank" rel=noopener href="https://www.cms.gov/About-CMS/Agency-Information/Aboutwebsite/Privacy-Policy">CMS Privacy Polciy</a></li>
+          <li><a target="_blank" rel=noopener href="https://www.cms.gov/vulnerability-disclosure-policy">CMS Vulnerability Disclosure</a></li>
```

Comment:
Text for this link should read "CMS/HHS Vulnerability Disclosure Policy" according to the Mural. I think someone updated the Mural since I created the ticket... 🙄